services/cards-consultation/src/main/java/org/opfab/cards/consultation/model/ArchivedCardConsultationData.java

```diff
@@ -81,9 +81,9 @@
     private List<String> groupRecipients;
     @Singular
     private List<String> entityRecipients;
-    @Singular("entityAllowedToRespond")
+    @Singular("entitiesAllowedToRespond")
     private List<String> entitiesAllowedToRespond;
-    @Singular("entityRequiredToRespond")
+    @Singular("entitiesRequiredToRespond")
```

Comment:
Let as it is , it must be in the singular and not plural  ,have a look to this article : 

 https://www.baeldung.com/lombok-builder-singular